onyo/lib/executors.py

```diff
@@ -15,7 +15,7 @@
 
 
 def exec_new_assets(repo: OnyoRepo, operands: tuple) -> tuple[list[Path], list[Path]]:
-    """Executor for the 'new_asset' operation
+    r"""Executor for the 'new_asset' operation
```

Comment:
Everywhere in this file the trailing `.` is missing. (At this point I comment this nitpick only because I did it above. Sorry for comment-spam)

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.